src/logger/logger.ts

```diff
@@ -0,0 +1,60 @@
+import moment from "moment-timezone"
+
+import { config } from "@root/config/config"
+
+import CloudWatchLogger from "./cloudwatch.logger"
+import { consoleLogger } from "./console.logger"
+import { Formatter, Loggable, Logger, LogMethod } from "./logger.types"
+
+const NODE_ENV = config.get("env")
+const useCloudwatchLogger = NODE_ENV === "prod" || NODE_ENV === "vapt"
```

Comment:
Sanity check here: should we use this for our staging env?

Reply:
good catch, somehow i missed teh `staging` when i copied (i checked teh deleted `logger.js`)